src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationHandler.cs

```diff
@@ -213,22 +220,20 @@ protected override string BuildChallengeUrl([NotNull] AuthenticationProperties p
 
         if (Options.UsePkce)
         {
-            var bytes = RandomNumberGenerator.GetBytes(32);
-            var codeVerifier = Microsoft.AspNetCore.Authentication.Base64UrlTextEncoder.Encode(bytes);
+            byte[] bytes = RandomNumberGenerator.GetBytes(32);
```

Comment:
I prefer it (and it's set up that way in [EditorConfig](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/1dcd2404d00226cd1f77e2af5f1686d2af4204f1/.editorconfig#L88), but I can turn it off if you want) when the type isn't obvious when you're reading the code in GitHub.

In this case it probably only changed because I copy-pasted it from somewhere where it wasn't using `var` already as part of this clean up, rather than an intentional change.